src/plugins/d3/examples/line/DataLabels.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
We do not use this file yet in storybook?

Reply:
the second example of a line graph in the [showcase](https://preview.gravity-ui.com/chartkit/347/?path=/story/plugins-d3-showcase--d-3-showcase-story)